### PR TITLE
Added ARM64 support to Sentinel and added ARM64 job to Travis-CI

### PR DESCRIPTION
**Pull request by odidev**
1. Added ARM64 architecture in .travis.yml
2. Updated 'embedded-consul' version to 2.2.0, 'consul-api' version to 1.4.5 and added groovy-all, slf4j-simple and slf4j-api dependencies for ARM64 support.
3. Updated grpc.version for 'io.grpc:protoc-gen-grpc-java' to 1.30.2, for ARM64 support.

Signed-off-by: odidev <odidev@puresoftware.com>

### CODE REVIEW

File: sentinel-extension/sentinel-datasource-consul/pom.xml [hunk 1]
```diff
@@ -15,11 +15,26 @@
...
+            <version>1.7.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.6.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
```
Comment (sczyh30) [inline sentinel-extension/sentinel-datasource-consul/pom.xml:35]: Why do we need these dependencies?

**Comment by odidev** [comment]
@sczyh30 
For adding ARM64 support, ‘embedded-consul’ version is upgraded, as 2.2.0 comes with ARM64 support. After upgrading ‘embedded-consul’ version from 2.0.0 to 2.2.0, both AMD64 and ARM64 jobs were failing due to dependency issues related to ‘slf4j’ and ‘groovy’, the solution to which is adding ‘groovy-all’, ‘slf4j-api’ and ‘slf4j-simple’ dependencies to pom.xml.

**Comment by odidev** [comment]
Hi Team,
Can you please review this PR?

**Comment by jasonjoo2010** [comment]
> @sczyh30
> For adding ARM64 support, ‘embedded-consul’ version is upgraded, as 2.2.0 comes with ARM64 support. After upgrading ‘embedded-consul’ version from 2.0.0 to 2.2.0, both AMD64 and ARM64 jobs were failing due to dependency issues related to ‘slf4j’ and ‘groovy’, the solution to which is adding ‘groovy-all’, ‘slf4j-api’ and ‘slf4j-simple’ dependencies to pom.xml.

I did a little test and a little analysis:

* `slf4j-api` is not necessary because it's introduced in embedded-consul
* `slf4j-simple` is not necessary. I know a warning will be thrown out but it doesn't matter and internal logs will be eaten. That's fine.
* `groovy-all` is not necessary. Instead, `groovy-xml` is the one needed in testing. Don't forget to mark it as test scope only.

**Comment by odidev** [comment]
@jasonjoo2010  @sczyh30 
Please review this and suggest if anything else is required.

**Comment by odidev** [comment]
@jasonjoo2010  @sczyh30 
Can you please review this PR?

**Comment by jasonjoo2010** [comment]
Hi, @odidev, sorry for the latency.

Now I have no further question on this PR. So do you have any further questions or different opinions on it, @sczyh30 ?

**Review by sczyh30 — APPROVED** [review]
LGTM

**Comment by sczyh30** [comment]
Nice work. Thanks for contributing!

**Comment by odidev** [comment]
@sczyh30 
Thank you for merging the PR. When can we expect the sentinel's new release with ARM64 support to the maven central?

**Pull request merged**